docs/src/man/getting_started.md

```diff
@@ -26,6 +26,20 @@ Throughout the rest of this tutorial, we will assume that you have installed the
     See [here](https://jupyter-client.readthedocs.io/en/stable/kernels.html) for information about location
     and specification of Jupyter kernels.
 
+
+## DataFrame Introduction
+ 
+A DataFrame is a table that can contain data of various types side-by-side. It is a two-dimensional data structure with three main components: rows, columns, and the data:
```

Comment:
we write data frame not DataFrame, as "data frame" means any object that is of `AbstractDataFrame` type. If we write DataFrame we write `DataFrame` to reference the `DataFrame` type specifically.